lblod/frontend-form-builder
# SFB-72: extend basic functionality with create and edit

## ID
 [SFB-72](https://binnenland.atlassian.net/browse/SFB-72)

 ## Description

The user wants to have a new interface where they want to see and edit code lists. The generic code lists may not be edited but the others can. I created a new page for showing the code lists and viewing them. And added a tag that will tell you if it is a public or private codelist.

When clicking the create button it create a new codelist with the default name . You have to change that name to proceed. On the edit page some buttons are enabled or disabled depending if the codelist is public or not.

 ## Type of change


 ## How to test

1. See the overview page are the tags correct
2. Click on a codelist and see if you can edit/delte it
3. Create your own codelist and see if you can edit/delete it

 ## Links to other PR's

 - https://github.com/lblod/frontend-form-builder/pull/84
